osmaxx/conversion/converters/converter_gis/bootstrap/bootstrap.py

```diff
@@ -23,22 +23,26 @@ def cut_area_from_pbf(pbf_result_file_path, extent_polyfile_path):
     subprocess.check_call(command)
 
 
+def polyfile_string_to_pbf(polyfile_string, pbf_out_path):
+    polyfile_path = os.path.join('/tmp', 'polyfile_extent.poly')
```

Comment:
Now that the generated `*.poly` file is only used locally (i.e. only inside this function), we could use a [`NamedTemporaryFile`](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile).